### PR TITLE
Adding php 81 for most extensions

### PR DESCRIPTION
**Pull request by shouze**
Related to https://github.com/brefphp/bref/pull/1007

Also related to https://github.com/brefphp/bref/pull/1008, waiting for upfix to be merged

### CODE REVIEW

**Comment by shouze** [comment]
:warning: @Nyholm please wait before merge, it's probably a good idea to update to 1.2.13 images once https://github.com/brefphp/bref/pull/1008 get merged

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # Change log
 
+## 0.12.0
+
+### Changed
+
+- Using bref/build-php-XX:1.2.12 as base image
```
Comment (shouze) [inline CHANGELOG.md:7]: Will be 1.2.13 after merge of https://github.com/brefphp/bref/pull/1008 which is an upfix

**Comment by Nyholm** [comment]
Cool cool. Let me know when it is ready to be reviewed.

**Comment by shouze** [comment]
> Cool cool. Let me know when it is ready to be reviewed.

This is ok now! :smiley:

**Comment by GrahamCampbell** [comment]
Can we support xdebug by compiling from master? https://gitlab.com/GrahamCampbell/PHP/-/blob/8.1/Dockerfile#L5-8

**Comment by shouze** [comment]
> Can we support xdebug by compiling from master? 

@GrahamCampbell do you mean like that https://github.com/brefphp/extra-php-extensions/pull/261/commits/3328679a29a5d80503bcea64644b4f2c258873bb? :wink:

**Comment by GrahamCampbell** [comment]
> do you mean like that

Yes. 🚢

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,29 @@
 # Change log
 
+## 0.12.0
```
Comment (Nyholm) [inline CHANGELOG.md:3]: I will change this to 0.11.15 before releasing. 
There are no BC breaks here.

**Review by Nyholm — APPROVED** [review]
Thank you

**Comment by mnapoli** [comment]
Wow xdebug support in PHP 8.1 already, that's crazy 🤯 

Amazing stuff here!

**Pull request merged**